### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kyliel**
Update some words and expressions. We may still need to restrucutre it a bit.

### CODE REVIEW

File: azure-webapp-maven-plugin/README.md [hunk 1]
```diff
@@ -155,9 +157,7 @@ Property | Required | Description | Version
     Users don't need to care about the deployment type in v2 configuration.
     Just configure the resources to deploy to the Web App.
     
-    It will use the zip deploy for fast and easy deploy.
-    But if the artifact(s) are war package(s), war deploy will be used.
-    Mix deploying war packages and other kinds of artifacts is not suggested and will cause errors.
+    By default, it will use the zip deploy which is fast and easy to complete the deployment. But if the artifact(s) are war package(s), war deploy will be used.Don't suggest mixing war packages and other kinds of artifacts.
```
Comment (jdneo) [inline azure-webapp-maven-plugin/README.md]: Add a link to zip deploy and war deploy.
- Zip Deploy: https://github.com/projectkudu/kudu/wiki/Deploying-from-a-zip-file
- War Deploy: https://github.com/projectkudu/kudu/wiki/Deploying-WAR-files-using-wardeploy

Comment (jdneo) [inline azure-webapp-maven-plugin/README.md]: Don't suggest mixing war packages and other kinds of artifacts. -> Mixing war packages and other kinds of artifacts are not suggested.

File: azure-webapp-maven-plugin/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 [![Maven Central](https://img.shields.io/maven-central/v/com.microsoft.azure/azure-webapp-maven-plugin.svg)](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.microsoft.azure%22%20AND%20a%3A%22azure-webapp-maven-plugin%22)
 
 The Maven Plugin for Azure App Service provides seamless integration into Maven projects, 
-and makes it easier for developers to deploy to different kinds of Azure Web Apps:
+and makes it easier for developers to deploy Java App to different kinds of Azure Web Apps:
```
Comment (jdneo) [inline azure-webapp-maven-plugin/README.md]: Java App -> Java applications

File: azure-webapp-maven-plugin/README.md (hunk 1 above)
Comment (andxu) [inline azure-webapp-maven-plugin/README.md]: Miss a space before `don't`

Comment (andxu) [inline azure-webapp-maven-plugin/README.md]: don't know the meaning of "mixing war packages and other kinds of artifacts"

**Comment by Flanker32** [comment]
README had been updated in another PR

**Pull request closed without merging**